packages/headless/src/controllers/facets/automatic-facet-generator/headless-automatic-facet-generator.test.ts

```diff
@@ -47,27 +50,12 @@ describe('automatic facets', () => {
 
   describe('#setOptions', () => {
     it('should dispatch #setOptions with valid options', () => {
-      expect(engine.actions).toContainEqual(setOptions(props.options));
```

Comment:
Moved them to `automatic-facet-slice-test`, better place to test the action validation itself directly.